src/documentation/pages/components/date-picker/examples/_base-date-picker.vue

```diff
@@ -0,0 +1,52 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base-date-picker" :tabs="exampleTabs">
+    div(style="max-width: 14rem;" slot="example")
+      chi-label(for='date') Date
+      chi-date-picker(id="date")
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+  data: () => {
+    return {
+      exampleTabs: [
+        {
+          active: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          id: 'htmlblueprint',
+          label: 'HTML blueprint',
+        },
+      ],
+      codeSnippets: {
+        webcomponent: `<chi-label for="date">Date</chi-label>
+<chi-date-picker id="date"></chi-date-picker>`,
```

Comment:
As for the `id` nomenclature we talked about, I would change each `id` in this pull request. 
Check this in each example.

```suggestion
<chi-date-picker id="example__base"></chi-date-picker>`,
```

Reply:
Sure, i can change this and going forward for other examples. The only concern is we were not doing this for the earlier examples and looks like they also have to be updated to mainatian consistency.